src/components/VariablesTable.vue

```diff
@@ -87,6 +91,15 @@
   const variableLike = ref<string>()
   const variableLikeDebounced = useDebouncedRef(variableLike, 1000)
   const offset = ref(0)
+  const page = computed({
+    get() {
+      return offset.value + 1
+    },
+    set(value: number) {
+      offset.value = value - 1
+    },
+  })
```

Comment:
this is confusing. Offset is used for the filter so I'd expect the offset to be `page * limit`. What's the reason for the get/set? The + 1 and - 1 is mysterious as well.

Reply:
Oh you're right, I forgot to take the limit into account; the reason is that the pager component doesn't work well with offsets and pages aren't something that exist API-side so we need a bridge to map the two.

Reply:
I think offset is computed. At least this is how I imagined it working when I built the pager

```
const offset = computed(() => page.value * limit)
```

Reply:
That makes sense - though we'll always need to subtract 1 from that offset if we're using human-readable page numbers (1 indexed instead of 0)

Reply:
Updated to this method btw

Reply:
correct, my example didn't account for that.